packages/terra-image/src/Image.jsx

```diff
@@ -143,28 +143,35 @@ class Image extends React.Component {
       src, variant, isFluid, alt, placeholder, height, width, onLoad, onError, fit, ...customProps
     } = this.props;
 
-    const imageClasses = cx([
+    const imageClassAttributes = [
       'image',
       fit,
       variant,
       customProps.className,
       { fluid: isFluid },
-    ]);
+    ];
+
+    if (placeholder && this.state.isLoading) {
+      imageClassAttributes.push('hidden');
+    }
+
+    const imageClasses = cx(imageClassAttributes);
+
     delete customProps.className;
     if (!this.state.isLoading) {
       objectFitImages(this.ImageRef.current);
     }
     if (placeholder) {
       if (this.state.isLoading) {
         return (
-          <div>
-            <div className={cx('hidden')}>{this.createImage(customProps, imageClasses)}</div>
-            <div>{placeholder}</div>
-          </div>
+          <React.Fragment>
```

Comment:
nitpick, can we use shorthand instead? `<>`

Reply:
I kind of prefer React.Fragment since it's easier on my old man eyes, but I'll go with the shorthand since you guys are gonna maintain terra-image more than I going forward.